iothub/device/tests/CertificateHelper.cs

```diff
@@ -35,7 +35,9 @@ public static void InstallCertificate(X509Certificate2 certificate, StoreLocatio
             }
             finally
             {
+#if !NETCOREAPP1_1
                 store.Close();
```

Comment:
context on why is this added?

Reply:
Close method is not supported in net core app 1.1.